woocommerce/woocommerce-gutenberg-products-block
# Unify the search bar layout across patterns

## What

Fixes #11408.

This PR unifies the style of the product search block across patterns. I marked with ⚠️ emoji the patterns that don't look okay. The reasons are:
- It is not possible to change the background of the input field: https://github.com/WordPress/gutenberg/issues/44671
- We need to improve the header patterns on mobile view (not related to this issue)

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add a new post.
2. Add the `Footer with Simple Menu and Cart`.
3. Save.
4. Ensure that the patterns look okay on desktop and mobile (and match with the screenshot).
5. Repeat 1-4 for each pattern in the table below.


## Screenshots or screencast




| Pattern Name | Desktop | Mobile |
|--------|--------|--------|
| Footer with Simple Menu and Cart |![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/62fbda44-8016-4408-992b-fdef5625dde7)|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/259777d7-6ded-43e8-ad35-400d7e95d5ec)|
| Footer with 3 Menus |  ![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/463e0413-60b7-411b-b251-f71888238dbe) |![locale local iPhone 12 Pro](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/712dca0a-51b2-4706-b9c4-83acf07ed773)|
| ⚠️ Centered Header Menu with Search | ![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/9f39877b-e062-4b68-92fe-3cb24588b22a)|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/53e3be1e-acd2-431e-87ca-ce318ab94a9d)|
| ⚠️ Essential Header Dark |![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/bed825bb-b6b8-4a75-b13a-cdb24acdf0a7) | ![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/b5cd1040-f54e-4555-815d-afc435545428)|
| ⚠️ Essential Header |![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/3d5415d1-63ac-46a5-92c9-5b3e35321fa0)| ![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/94235172-3487-49db-ad6f-f0b5da566e41) |
| ⚠️ Header Large Dark |![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/8c70c0b7-80fe-4508-9bb1-deede3219354) | ![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/f234e718-8216-4254-a163-cd7884cfaaa2)|
| ⚠️ Header Large | ![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/69872bf5-c038-4ede-a9fa-098ccb8216a5)| ![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/00bb1d0c-0660-4a9f-a221-a09e38ce9a8e)|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
